### PR TITLE
fix(simple-dialogs): add demos back

### LINKED ISSUE #1632: Simple Dialogs Demo
Simple Dialogs
Add all previous 
Add new demo for resizable

### PR DESCRIPTION
**Pull request by JulieMarie**
## Description
Added missing demos for simple dialogs

### What's included?

- Added missing demos for simple dialogs

#### Test Steps

- [ ] `npm run serve`
- [ ] Components -> Simple Dialogs -> Examples

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

Closes #1632 

##### Screenshots or link to StackBlitz/Plunker

### CODE REVIEW

File: src/app/content/components/component-demos/dialogs/demos/dialogs-demo-resizable/dialogs-demo-resizable.component.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import { Component, Renderer2, Inject, RendererFactory2 } from '@angular/core';
```
Comment (JulieMarie) [inline src/app/content/components/component-demos/dialogs/demos/dialogs-demo-resizable/dialogs-demo-resizable.component.ts]: I was supposed to include a "simple" resizable dialog, but this is anything but simple and I think it defeats the purpose of these demos.  What's here is almost complete, but still doesn't work and has no dialog in the dialog box and it's nothing "simple".  I would suggest to not include.

Comment (christianmemije) [reply src/app/content/components/component-demos/dialogs/demos/dialogs-demo-resizable/dialogs-demo-resizable.component.ts]: Nooooo :(

File: src/app/content/components/component-demos/dialogs/demos/dialogs-demo-basic/dialogs-demo-basic.component.ts [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 
 @Component({
   selector: 'dialogs-demo-basic',
   styleUrls: ['./dialogs-demo-basic.component.scss'],
   templateUrl: './dialogs-demo-basic.component.html',
-  preserveWhitespaces: true,
+  animations: [slideInDownAnimation],
 })
 export class DialogsDemoBasicComponent {
+  @HostBinding('@routeAnimation') routeAnimation: boolean = true;
```
Comment (christianmemije) [inline src/app/content/components/component-demos/dialogs/demos/dialogs-demo-basic/dialogs-demo-basic.component.ts]: Not necessary

File: src/app/content/components/component-demos/dialogs/demos/dialogs-demo-basic/dialogs-demo-basic.component.html [hunk 3]
```diff
@@ -1 +1,3 @@
-<button mat-raised-button color="primary" (click)="openAlert()" class="text-upper">Open Alert</button>
\ No newline at end of file
+<button mat-button color="accent" (click)="openAlert()" class="text-upper push-right-sm">Open Alert</button>
```
Comment (christianmemije) [inline src/app/content/components/component-demos/dialogs/demos/dialogs-demo-basic/dialogs-demo-basic.component.html]: Probably separate these into 3 demos, and use the first one as the hero.

Comment (JulieMarie) [reply src/app/content/components/component-demos/dialogs/demos/dialogs-demo-basic/dialogs-demo-basic.component.html]: Yep, I agree.

**Pull request merged**